staff/pep-delbano/app-template/profile.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Profile</title>
+</head>
+<body>
+    <header>
+        <h1>My Name</h1>
+        <q>motivational quote</q>
+        <p>Introduction and some info about my goals</p>
+        <img src="#" alt="portrait picture">
+    </header>
+    <nav><!--contains a list as it'll be a side nav-->
+        <ul>
+            <li><a href="#">EXPERIENCE</a></li>
+            <li><a href="#">SKILLS</a></li>
+            <li><a href="#">REFENCES</a></li>
+            <li><a href="#">PROJECTS</a></li>
+        </ul>
+    </nav>
+
+    <article>
+        <h2>EXPERIENCE</h2>
+        <section>
+            <h3>COMPANY</h3>
+            <p>some duties info</p>
+        </section>
+        <section>
+                <h3>COMPANY</h3>
+                <p>some duties info</p>
+            </section>
+            <section>
+                    <h3>COMPANY</h3>
+                    <p>some duties info</p>
+                </section>
+    </article>
+    <article>
+        <h2>SKILLS</h2>
+        <section>
+            <h3>HTML5</h3>
+            <progress value="70" max="100"></progress>
+        </section>
+        <section>
+            <h3>CSS3</h3>
+            <progress value="70" max="100"></progress>
+        </section>
+        <section>
+            <h3>JS</h3>
+            <progress value="60" max="100"></progress>
+        </section>
+    </article>
+
+    <article>
```

Comment:
same here and in the other next blocks... may you just split blocks with sections instead?